### PR TITLE
Add Types Definition

### PR DESCRIPTION
**Pull request by tsullivan**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
```
Comment (spalger) [inline index.d.ts]: Please remove this

**Review by spalger — APPROVED** [review]
One nit, but LGTM otherwise

**Comment by spalger** [comment]
Feel free to ignore the node 6 error, I've removed that from the travis.yml file

File: index.d.ts (hunk 1 above)
Comment (tsullivan) [reply index.d.ts]: b77da6a

**Pull request merged**